### ISSUE TITLE
Excluded cases

### ISSUE THREAD

**Issue by Mark-Torrance** [Mark-Torrance/Snod_EN_analysis#2]
So that I don't forget...

The data in `snod_EN_keydata_2.Rdata` do not include trials in which the subject edited their response in any way. So any trials where cursor keys or backspace were used are excluded (there was no other editing functionality - no mouse for example).

All other trials are in.

**Comment by mark-andrews** [comment]
Right. I was wondering about that. In general, is the data that you want to analyse only that where there was no editing of this kind? It certainly simplifies things to exclude these cases. But presumably editing happens on a relatively large number of trials and presumably edits also tells as much as e.g. keystroke latencies.

**Comment by Mark-Torrance** [comment]
I think it's a case of picking your battles. We also need to make a decision about non-modal responses. For the "main paper" I just analysed trials where both the name and the spelling were modal. i.e. they gave the "correct" name and spelling. This makes the results much easier to interpret. I really don't know what you'd do with IKIs when there's editing, and I think misspelling is also a special case. Jens might well be interested in spelling and typos, though. He / we are working with Rianne who is doing stuff on this.

**Comment by Mark-Torrance** [comment]
In a rush, but note to self - explain the "whitespace issue"

**Comment by mark-andrews** [comment]
Agreed

On Sat, 2019-01-26 at 11:58 -0800, Mark Torrance wrote:
> I think it's a case of picking your battles. We also need to make a
> decision about non-modal responses. For the "main paper" I just
> analysed trials where both the name and the spelling were modal. i.e.
> they gave the "correct" name and spelling. This makes the results
> much easier to interpret. I really don't know what you'd do with IKIs
> when there's editing, and I think misspelling is also a special case.
> Jens might well be interested in spelling and typos, though. He / we
> are working with Rianne who is doing stuff on this.
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
DISCLAIMER: This email is intended solely for the addressee. It may contain private and confidential information. If you are not the intended addressee, please take no action based on it nor show a copy to anyone. In this case, please reply to this email to highlight the error. Opinions and information in this email that do not relate to the official business of Nottingham Trent University shall be understood as neither given nor endorsed by the University. Nottingham Trent University has taken steps to ensure that this email and any attachments are virus-free, but we do advise that the recipient should check that the email and its attachments are actually virus free. This is in keeping with good computing practice.
